opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp

```diff
@@ -636,7 +650,8 @@ namespace Opm
     {
         const RestartConfig& restartConfig = eclipseState_->getRestartConfig();
         const int reportStepNum = timer.reportStepNum();
-        std::vector<data::CellData> cellData = detail::getCellData( phaseUsage_, physicalModel, restartConfig, reportStepNum );
+        std::vector<data::CellData> cellData = detail::getCellData( phaseUsage_, physicalModel, restartConfig, reportStepNum,
+                                                                    output_ && parallelOutput_->isIORank() );
```

Comment:
Without a  name the bool parameter is a bit misleading.  It only switches the logging on or of. I made  a change to make this more clear. The cellData is always populated with the local values.